src/preferences/dialog/dlgpreflibrarydlg.ui

```diff
@@ -138,10 +138,16 @@
         </property>
        </widget>
       </item>
+      <item row="1" column="0">
+       <widget class="QCheckBox" name="checkBox_SeratoMetadataExport">
+        <property name="text">
+         <string>Write Serato Metadata to files</string>
```

Comment:
Write Serato Metadata to files (experimental)

Reply:
Did @Be-ing ask you to remove it earlier? I disagree. Such a warning should not only appear in a popup dialog, hidden in a paragraph of text.